integration_test/third_party_apps_data/applications/nvml/centos_rhel/install

```diff
@@ -32,16 +32,23 @@ case $DEVICE_CODE in
         ;;
     *)
         # Installing latest version of NVIDIA CUDA and driver
-        DRIVER_VERSION=535.104.05
+        # Data Center/Tesla drivers and CUDA are released on different schedules;
+        # normally we install the matching versions of driver and CUDA 
+        # ($DRIVER_VERSION == $CUDA_BUNDLED_DRIVER_VERSION); due to https://github.com/NVIDIA/open-gpu-kernel-modules/issues/550
+        # we install a newer version of the driver
+        DRIVER_VERSION=535.129.03
         CUDA_VERSION=12.2.2
+        CUDA_BUNDLED_DRIVER_VERSION=535.104.05
         ;;
 esac
 
 echo "Installing NVIDIA CUDA $CUDA_VERSION with driver $DRIVER_VERSION"
```

Comment:
Does this need to be updated to also reflect `$CUDA_BUNDLED_DRIVER_VERSION`? Or add another log message?

Reply:
Done! Now using two logs to show the driver version, and CUDA version and the installer name containing the `$CUDA_BUNDLED_DRIVER_VERSION`.